tests/phpunit/tests/fonts/font-face/wpFontFaceResolver/getFontsFromThemeJson.php

```diff
@@ -37,20 +37,59 @@ public function test_should_return_all_fonts_from_theme() {
 		$this->assertSame( $expected, $actual );
 	}
 
+	public function test_should_return_all_fonts_from_all_theme_origins() {
```

Comment:
Needs to have the ticket annotation to identify which ticket it relates to for the bugfix.

```suggestion
	/**
	 * @ticket 60605
	 */
	public function test_should_return_all_fonts_from_all_theme_origins() {
```